components/eam/cime_config/config_component.xml

```diff
@@ -81,13 +81,14 @@
       <value compset="_EAM%MMF[12XO]"         >-crm_nx 64 -crm_nx_rad 16 -crm_ny 1 -crm_ny_rad 1 </value>
       <value compset="_EAM%MMF*-AQUA"         >-aquaplanet </value>
       <value compset="_EAM%MMF*-RCE"          >-aquaplanet -rce </value>
+      <value compset="_EAM%MMF[12XO]-SCAM"    >-scam </value>
```

Comment:
I don't think this will work for MMFOMP or MMFXX options. I think we just need an asterisk after "[12XO]", but I'm not 100% sure that will work.

Reply:
At first, I used '_EAM%MMF*-SCAM' but it didn't work. I am not 100% sure how the wildcard works in xml. I will test MMF[12XO]*-SCAM and see it if works for MMFXX and MMFOMP.

Reply:
Also, per our slack convo, I think we should use "SCM" instead of "SCAM"

Reply:
Maybe you need a wildcard for each character. It's fine to just add multiple lines to avoid the wildcard issue.

Reply:
MMF[12XO]*-SCM worked.